Berksfile.lock

```diff
@@ -1,56 +1,55 @@
 DEPENDENCIES
   apt
+  fix-ipv6-kitchen-dokken
+    path: test/fixtures/cookbooks/fix-ipv6-kitchen-dokken
   site-reviewtypo3org
     path: .
     metadata: true
   site-reviewtypo3org_test
     path: test/fixtures/cookbooks/site-reviewtypo3org_test
 
 GRAPH
-  apache2 (3.2.2)
+  apache2 (4.0.0)
   apt (2.9.2)
-  aws (4.2.0)
-    ohai (>= 4.0)
-  backuppc (1.0.1)
-    apache2 (>= 0.0.0)
+  aws (7.2.2)
+  backuppc (1.0.2)
+    apache2 (~> 4.0)
     apt (>= 0.0.0)
     htpasswd (>= 0.0.0)
     ssl_certificates (>= 0.0.0)
     sudo (>= 0.0.0)
     user (>= 0.0.0)
-  build-essential (7.0.3)
-    compat_resource (>= 12.16.3)
-    mingw (>= 1.1)
-    seven_zip (>= 0.0.0)
+  build-essential (2.2.4)
```

Comment:
Here its still 2.x instead of 8 required by the mysql-ruby gem in the gerrit cookbook, shouldn't this be gone after removing the proxy?

Reply:
you're right.. I completely ignored the `Berksfile.lock` as had more changes in the meantime. That file should also be updated, of course.